shared/utils/urls.ts

```diff
@@ -83,10 +87,12 @@ export function sanitizeUrl(url: string | null | undefined) {
   }
 
   if (
-    !isUrl(url) &&
-    !url.startsWith("/") &&
-    !url.startsWith("#") &&
-    !url.startsWith("mailto:")
+    (!isUrl(url) &&
+      !url.startsWith("/") &&
+      !url.startsWith("#") &&
+      !url.startsWith("mailto:")) ||
+    url.startsWith("javascript://") ||
+    url.startsWith("file://")
```

Comment:
```suggestion
    url.startsWith("javascript:") ||
    url.startsWith("file:")
```

Javascript does not require the `//`